docs/using-the-jdbc-driver/using-plugins/UsingTheFailoverPlugin.md

```diff
@@ -30,7 +30,7 @@ In addition to the parameters that you can configure for the underlying driver,
-| `keepSessionStateOnFailover`           | Boolean |                                        No                                        | This parameter will allow connections to retain the session state after failover. When keepSessionStateOnFailover is set to false, connections will need to be reconfigured as seen in the example [here](./../../../examples/AWSDriverExample/src/main/java/software/amazon/PgFailoverSample.java). When this parameter is true, the autocommit and readOnly values will be kept. This parameter is only necessary when the session state must be retained and the connection cannot be manually reconfigured by the user. <br><br> **Please note:** this parameter will not be able to fully restore the connection session state, as it will only save the autocommit and readOnly values.                                                                                                                                                                                                                                                                                                                        | `false`                                                                                                                                                                                             |
+| ~~`keepSessionStateOnFailover`~~       | Boolean |                                        No                                        | This parameter is no longer available and, if specified, it will be ignored by the driver. See [Session State](../SessionState.md) for more details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | `false`                                                                                                                                                                                             |
```

Comment:
```suggestion
| ~~`keepSessionStateOnFailover`~~       | Boolean |                                        No                                        | This parameter is no longer available. If specified, it will be ignored by the driver. See [Session State](../SessionState.md) for more details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | `false`                                                                                                                                                                                             |
```